src/assets/scripts/client/airport/MapCollection.js

```diff
@@ -0,0 +1,181 @@
+import _isArray from 'lodash/isArray';
+import BaseCollection from '../base/BaseCollection';
+import MapModel from './MapModel';
```

Comment:
We generally try to sort these by the file path instead of the object. When we import multiple things from a given file, it saves us from having to reshuffle them all around. But otherwise the order here is good-- lodash, then classes, then constants and such.